### PR TITLE
Feature/57,58 function commons

### PR DESCRIPTION
**Pull request by dk1844**
@jakipatryk @benedeki

### CODE REVIEW

File: ultet/src/main/scala/za/co/absa/ultet/model/function/FunctionGrant.scala [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package za.co.absa.ultet.model.function
+
+case class FunctionGrant() extends FunctionEntry () {
```
Comment (jakipatryk) [inline ultet/src/main/scala/za/co/absa/ultet/model/function/FunctionGrant.scala:18]:
```suggestion
case class FunctionGrant() extends FunctionEntry {
```

File: build.sbt [hunk 2]
```diff
@@ -102,4 +102,19 @@ lazy val faDBExamples = (project in file("examples"))
...
 
+lazy val ultet = (project in file("ultet"))
+  .configs(IntegrationTest)
+  .settings(
+    name := "examples",
+    libraryDependencies ++= examplesDependencies(scalaVersion.value),
+    Test / parallelExecution := false,
+    (Compile / compile) := ((Compile / compile) dependsOn printScalaVersion).value, // printScalaVersion is run with compile
+    publish / skip := true
+  ).dependsOn(faDbCore, faDBSlick)
```
Comment (lsulak) [inline build.sbt:113]: no dependencies

File: build.sbt [hunk 3]
```diff
@@ -102,4 +102,19 @@ lazy val faDBExamples = (project in file("examples"))
     jacocoExcludes := commonJacocoExcludes
   )
 
+lazy val ultet = (project in file("ultet"))
+  .configs(IntegrationTest)
+  .settings(
+    name := "examples",
```
[the remark this answers is not in the document]
Comment (dk1844) [reply build.sbt]: removed, thanks

File: build.sbt (hunk 2 above)
Comment (dk1844) [reply build.sbt]: removed, thanks

File: ultet/src/main/scala/za/co/absa/ultet/model/function/FunctionGrant.scala (hunk 1 above)
Comment (dk1844) [reply ultet/src/main/scala/za/co/absa/ultet/model/function/FunctionGrant.scala]: thanks, fixed

File: ultet/src/main/scala/za/co/absa/ultet/dbitems/DBItem.scala [hunk 4]
```diff
@@ -16,7 +16,7 @@
 
 package za.co.absa.ultet.dbitems
 
-import za.co.absa.ultet.sqlentry.SQLEntry
+import za.co.absa.ultet.model.SQLEntry
 
 trait DBItem {
   def sqlEntry: SQLEntry
```
Comment (lsulak) [inline ultet/src/main/scala/za/co/absa/ultet/dbitems/DBItem.scala:22]: this will be a seq, can you change it?

Comment (dk1844) [reply ultet/src/main/scala/za/co/absa/ultet/dbitems/DBItem.scala]: sure, done.

**Review by lsulak — APPROVED** [review]

**Review by jakipatryk — APPROVED** [review]

**Pull request merged**